draft-ietf-httpbis-priority.md

```diff
@@ -130,8 +130,8 @@ consideration and guidance about how servers might act upon signals.
 
 {::boilerplate bcp14-tagged}
 
-The terms Dictionary, sf-boolean, sf-dictionary, and sf-integer are imported from
-{{!STRUCTURED-FIELDS=RFC8941}}.
+The terms Boolean, Dictionary, and Integer, along with the sf-dictionary rule,
```

Comment:
I don't think that you want to say that RFC 8941 defines the term "Boolean".  Maybe instead "This document uses the Boolean, Dictionary, and Integer types from {{!SF}} along with the `sf-dictionary` ABNF rule."

Reply:
Thanks. "Boolean" is so generic, it was part of the reason why I gravitated to `sf-boolean` in prose in the first place.

Reply:
In prose, you can try "Boolean Structured Type" or similar.

Reply:
The pattern I've seen so far is people saying "Structured Fields Dictionary" or "Structured Fields List". So "Structured Fields Boolean" would be consistent with that but its a bit wordy. Perhaps settling on a common acronym of `SF` in front of these would work.

Reply:
Applying this to "SF Member" would also be help in specs that need to talk generically about the things in List or Dictionary without wanting to have to make up a term of their own.